circle.yml

```diff
@@ -4,13 +4,13 @@ jobs:
 
   lint:
     working_directory: /work
-    docker: [{image: 'docker:18.03-git'}]
+    docker: [{image: 'docker:18.09.0-git'}]
     steps:
       - checkout
       - setup_remote_docker:
-            version: 18.03.1-ce
-            reusable: true
-            exclusive: false
+          version: 18.09.0
```

Comment:
Looks like the Circle-CI images did not yet update the version-scheme; the Docker versions no longer have a `-ce` / `-ee` suffix 

```
Allocating a remote Docker Engine
failed to create host: rpc error: code = Unknown desc = image docker-18.09.0 is not supported
```

/cc @dnephin perhaps they should have an alias?